crates/l2/sequencer/proof_coordinator.rs

```diff
@@ -213,8 +214,9 @@ impl ProofCoordinatorState {
     }
 }
 
+#[derive(Clone)]
 pub enum ProofCordInMessage {
-    Listen { listener: TcpListener },
+    Listen { listener: Arc<TcpListener> },
```

Comment:
I'm failing to see how this relates to the PR. Is this a separate fix for a separate problem?